spec/features/user_edit_spec.rb

```diff
@@ -1,22 +1,48 @@
 # These are tests for future features for editing your user page
-# require 'rails_helper'
-#
-# feature 'edit profile',%{
-#    As a user
-#    I want to edit my profile
-#    So that I can stand out on this fancy site
-# } do
-#
-#   # scenario 'user edits profile page' do
-#   #   visit '/users/edit'
-#   #   click_button 'Update'
-#   #
-#   #   expect(page).to have_content("Current password can't be blank")
-#   # end
-#     #
-#     # scenario 'user edits profile page' do
-#     #   visit '/users/edit'
-#     #
-#     #   expect(page).to have_content("Edit User")
-#     # end
-# end
+require 'rails_helper'
+
+feature 'edit profile' do
+  let!(:user) { FactoryBot.create(:user) }
+
+  scenario 'vistor tries to navigate to an edit page' do
+    visit '/users/edit'
+
+    expect(page).to have_content('Email')
+    expect(page).to have_content('Home')
+    expect(page).to have_content('Sign In')
+    expect(page).to have_content('Sign Up')
+    expect(page).to have_content('Password')
+  end
+
+  scenario 'user edits their account' do
+    visit user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: 'Password'
+    click_button 'Sign In'
+    visit edit_user_registration_path
+    expect(page).to have_content 'Edit'
+    expect(find_field('Email').value).to eq user.email
+    expect(find_field('Username').value).to eq 'jsmith'
+  end
+
+  scenario 'user edits their account but does not enter current password' do
+    visit user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: 'Password'
+    click_button 'Sign In'
+    visit edit_user_registration_path
+    click_button 'Update'
+    expect(page).to have_content 'Current password can\'t be blank'
```

Comment:
By "current password" do you mean correct password?  Current password makes it sound like it's frequently changing.

Reply:
Current Password is a field on the edit page that must be filled in to make profile changes